### ISSUE TITLE
Reformatting grid tables seems to add spaces every time

### ISSUE THREAD

**Issue by bavovanachte** [shuGH/vscode-table-formatter#3]
First of all, thanks for this plugin. It really helps a lot.
I'm using it for grid tables in rst, but there is a bit of an annoyance

This is the table I use as input: 

```
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
| Methods                          | ASIL A   | ASIL B   | ASIL C         | ASIL D   | Evidence or motivation                                      |
+==================================+==========+==========+================+==========+=============================================================+
|                                  |          |          | This Project   |          |                                                             |
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
| 1a Requirements-based test [a]   | ``++``   | ``++``   | ``++``         | ``++``   | bla                                                         |
|                                  |          |          |                |          | bla                                                         |
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
| 1b Interface test                | ``++``   | ``++``   | ``++``         | ``++``   | mock text                                                   |
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
| 1c Fault injection test [b]      | ``+``    | ``+``    | ``+``          | ``++``   | not applicable                                              |
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
| 1d Resource usage test [c]       | ``+``    | ``+``    | ``+``          | ``++``   | not applicable                                              |
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
| 1e Back-to-back comparison       | ``+``    | ``+``    | ``++``         | ``++``   | Example text                                                |
| test between model and code,     |          |          |                |          | More text                                                   |
| if applicable [d]                |          |          |                |          |                                                             |
+----------------------------------+----------+----------+----------------+----------+-------------------------------------------------------------+
```

Every time I execute the format command, it will make all the columns a bit wider. I would expect the plugin to 
- Leave the columns that are OK alone
- Shrink the last column, as it has excess whitespace.

Thanks,

Bavo

**Comment by shuGH** [comment]
Thanks for reporting the issue. and thanks for using.
Sorry for the late reply.

This problem was reproduce in my local.
I will probably fix it in the back one week.
Please wait for a while.

Thanks

**Comment by shuGH** [comment]
@bavovanachte 

Hi 
I fixed this problem at 1.0.3 and released.
Please check it.
If this problem is not reproduce close this issue ticket.

Thanks

**Comment by bavovanachte** [comment]
:+1: Tested now, and it works. Thanks a lot!
